client/resource_group/controller/controller.go

```diff
@@ -161,6 +169,9 @@ func (c *ResourceGroupsController) Start(ctx context.Context) {
 	c.initRunState()
 	c.loopCtx, c.loopCancel = context.WithCancel(ctx)
 	go func() {
+		c.run.responseDeadline = time.NewTimer(time.Second)
+		c.run.responseDeadline.Stop()
```

Comment:
remove this line?

Reply:
ref https://github.com/tikv/pd/pull/6063#discussion_r1122573946, we new a Timer but don't need it send Time channel at beginning.